src/ui/components/pages/PermissionsSettings/components/list/List.tsx

```diff
@@ -23,7 +23,7 @@ export class List extends React.PureComponent<IProps> {
 
     return (
       <div className={styles.permissionList}>
-        {originsNames.map(name => (
+        {originsNames.sort().map(name => (
```

Comment:
Because the sequence of sources is incorrectly displayed in this list due to the fact that sometimes the reference to the object is lost.

`const list = { 'b': 0, 'c': 1, 'a': 2 }
Object.keys(list) => ['b', 'c', 'a']`

But when the reference to the object is lost, an object with sorted fields by default is obtained instead:

`Object.keys({ 'a': 2, 'b': 0, 'c': 1 }) => ['a', 'b', 'c']`

Reply:
Not sure what you mean when you say "reference to the object is lost". How is it lost?

Reply:
When creating the `origins` object, the keys are written in the order that comes from the backend. Sometimes this object sorts its keys, most likely this is due to the loss of the original structure. I do not know why the reference to the object was lost, it only happens in chromedriver.